__init__.py

```diff
@@ -91,6 +93,12 @@ def __init__(self):
         self.query_cache = {}
         self.location_translations = {}
 
+        if api_key and not use_proxy:
+            self.owm = OWM(api_key)
```

Comment:
I wonder if `local_owm` or similar would be more descriptive?

Reply:
yes, I was also thinking about that or calling the other owm owm_wrapper